Sources/Subs-Compat.php

```diff
@@ -176,4 +176,208 @@ function smf_crc32($number)
 	}
 }
 
+if (!is_callable('random_int')) {
```

Comment:
This fallback function won't work unless we also include a fallback for `random_bytes()`, which would mean including most or all of Paragon's entire random_compat library.